lib/apiGateway/validate.test.js

```diff
@@ -1915,7 +1915,7 @@ describe('#validateServiceProxies()', () => {
       expect(() => serverlessApigatewayServiceProxy.validateServiceProxies()).to.not.throw()
     })
 
-    it('should not throw error if the "hashKey" is object and has both "pathParam" and "attributeType" properties', () => {
+    it('should not throw error if the "hashKey" is object and has both "keyName" and "attributeType" properties', () => {
```

Comment:
There's already a ` it('should not throw error if the "hashKey" is object and has both "pathParam" and "attributeType" properties', () => {` on line 1876 so I repurposed this test.